src/Twig/GlobalVariables.php

```diff
@@ -111,4 +111,12 @@ private function getCodeAction($code, $action)
 
         return [$action, $code];
     }
+
+    /**
+     * @return string
+     */
+    public function getDefaultMosaicBackground()
+    {
+        return $this->adminPool->getContainer()->getParameter('sonata.admin.configuration.default_mosaic_background');
```

Comment:
So I must use the `$container`variable even if it's deprecated?
```
    /**
     * @var ContainerInterface
     *
     * @deprecated Since version 3.5, will be removed in 4.0.
     * NEXT_MAJOR : remove this property
     */
    protected $container;
```

Reply:
You should use neither, do not access the container through a method or the
through a variable. Instead, please inject the parameter through the constructor

Reply:
Done. Thanks for advise.